apps/docs/svelte.config.js

```diff
@@ -1,25 +1,9 @@
-import adapter from '@sveltejs/adapter-auto';
+import adapter from '@sveltejs/adapter-static';
 import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
 
-/** @type {import('@sveltejs/kit').Config} */
 const config = {
-	// Consult https://kit.svelte.dev/docs/integrations#preprocessors
-	// for more information about preprocessors
 	preprocess: vitePreprocess(),
-
-	kit: {
-		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
-		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
-		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
-		adapter: adapter()
-	},
-
-	onwarn: (warning, handler) => {
-		const { code } = warning;
-		if (code === 'css-unused-selector') return;
```

Comment:
Do you remember why you added this , and do you know if it is stil lneeded?

See https://github.com/Greater-London-Authority/ldn-viz-tools/pull/494 and https://github.com/Greater-London-Authority/ldn-viz-tools/pull/494/commits/e8133170e1f77b7a56fa518168831b9a4b2801c8